tests/005_spec.lua

```diff
@@ -232,7 +238,7 @@ vim.api.nvim_set_hl(0, "diffRemoved", { fg = "#e06c75" })
 
         local file = io.open(require("onedarkpro.config").config.cache_path .. "onedark.lua", "r"):read("*all")
         -- file:close()
-        assert.are.same(file, expected)
+        assert.are.same(expected, file)
```

Comment:
@olimorris this isn't *needed* for anything, but it makes the test output "correct" / clearer. Before, it would say "received <and here would be the contents of the expected var in the test>, expected <and here would be the generated cache file contents>", which was backwards.

Reply:
See the output here to see what I mean: https://github.com/olimorris/onedarkpro.nvim/actions/runs/3256807886/jobs/5347645704.

This is from before I changed the arg order and fixed the test. The lines with `php` in them should be in the "passed in" section, not the "expected section".